apps/condo/domains/ai/constants.js

```diff
@@ -74,6 +76,28 @@ const FLOW_META_SCHEMAS = {
             },
         },
     },
+    [NEWS_REWRITE_TEXT_FLOW_TYPE]: {
+        input: {
+            type: 'object',
+            properties: {
+                inputInitiator: { enum: ['header', 'body'] },
+                currentHeaderNews: { type: 'string' },
+                currentBodyNews: { type: 'string' },
+
+                newsType: { type: 'string' },
+                newsActualUntil: { type: 'string' },
```

Comment:
Please use field names from `NewsItem` model. This field is called `validBefore` if im not mistaken